src-tauri/src/main.rs

```diff
@@ -145,7 +152,7 @@ async fn main() -> Result<()> {
                 frontend_info_logger
             ],
         )
-        .build(tauri::generate_context!())
+        .build(ctx)
         .expect("error while running tauri application");
```

Comment:
that's why, we replaced this one with the previously generated context :)
I tested this locally, both in DEV and PROD, it worked.